src/constructors.jl

```diff
@@ -53,6 +53,8 @@ function Taylor1(x::T, order::Int) where {T<:Number}
     v[1] = x
     return Taylor1(v, order)
 end
+Taylor1(x::T) where {T<:Number} = Taylor1(x, 0)
+Taylor1{T}(x::S) where {T<:Number, S<:Union{Real, Complex}} = Taylor1([convert(T,x)], 0)
```

Comment:
I'm not sure if `Union{Real, Complex}` should rather be `NumberNotSeries`, or not. In any case, for the current purposes, I think this should be the only methods added.

Reply:
Thank you for the suggestion; I've changed it now to `NumberNotSeries`, but moved the constructor definition towards the end of the file in order to be able to use it